.github/workflows/abidiff.yml

```diff
@@ -0,0 +1,50 @@
+name: AWS-LC ABI Diff
+on:
+  pull_request:
+    branches: [ '*' ]
+env:
+  DOCKER_BUILDKIT: 1
+  GOPROXY: https://proxy.golang.org,direct
+jobs:
+  crypto:
+    name: libcrypto
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          repository: aws/aws-lc
+          ref: ${{ github.ref }}
+          path: ${{ github.workspace }}/next
+      - uses: actions/checkout@v3
+        with:
+          repository: aws/aws-lc
+          ref: ${{ github.base.ref }}
+          path: ${{ github.workspace }}/previous
+      - name: Build Docker Image
+        working-directory: ${{ github.workspace }}/next/.github/docker_images/abidiff
+        run: |
+          docker build -t abidiff .
+      - name: Perform ABI Diff
+        run: |
+          docker run -v ${{ github.workspace }}/previous:/previous -v ${{ github.workspace }}/next:/next -- abidiff crypto
+  ssl:
```

Comment:
This is very minor, just curious: does this result in build AWS-LC 4 times? next/previous each for libcrypto and libssl?

Reply:
💰💰💰

Would we need to double this to 8 builds to get 4 CI tasks:
* Crypto API Change
* Crypto Incompatible API Change
* SSL API Change
* SSL Incompatible API Change

Reply:
💰💰💰

Would we need to double this to 8 builds to get 4 CI tasks:
* Crypto API Change
* Crypto Incompatible API Change
* SSL API Change
* SSL Incompatible API Change

Reply:
I think I found a way to combine them such that the overall action will correctly fail, but it will still diff libssl if the libcrypto step fails, thus allowing the sharing of the built artifacts.